src/js/claims-status/components/AddFilesForm.jsx

```diff
@@ -90,12 +90,12 @@ class AddFilesForm extends React.Component {
         <div>
           <ErrorableFileInput
             errorMessage={this.getErrorMessage()}
-            label={<h5>Select files to upload</h5>}
+            label={<h5 className="claims-upload-input-title">Select files to upload</h5>}
```

Comment:
I don't think this should be an `h5`, that doesn't really make sense in a label

Reply:
Good point @jbalboni, additional style rules have been provided to `claims-upload-input-title` to mimic the `<h5>` appearance.